src/content/enterprise/index.md

```diff
@@ -160,7 +160,7 @@ Here are some of the enterprise applications that have been deployed to the publ
 
 ### Credentials and certifications {#credentials}
 
-- [Utah Counties](http://www.utahcounty.gov/Dept/ClerkAud/DigitalCertCopy.html) _issuing digital marriage certificates on Ethereum_
+- [Utah Counties](https://web.archive.org/web/20210731052556/https://www.utahcounty.gov/Dept/ClerkAud/DigitalCertCopy.html) _issuing digital marriage certificates on Ethereum_
```

Comment:
```suggestion
(https://web.archive.org/web/20210731052556/https://www.utahcounty.gov/Dept/ClerkAud/DigitalCertCopy.html) _issuing digital marriage certificates on Ethereum_
```

I think this should be removed if we are using a wayback link. Given it doesnt seem this is currently being used after looking at their page, we should remove this.